### PR TITLE
[AF-1519] Added a second version of takari

### PR DESCRIPTION
**Pull request by desmax74**
We add a second version to use a working old version with settings.xml with authentication for Jenkins and a new one inside the compiler   @domhanak @jhrcek @porcelli @mareknovotny @ederign

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -216,7 +216,8 @@
     <gwt.compiler.localWorkers>1</gwt.compiler.localWorkers>
     <!-- property for productisation to know the last released version -->
     <latestReleasedVersionFromThisBranch>notYetReleased</latestReleasedVersionFromThisBranch>
-    <version.io.takari.maven.plugins>1.13.5</version.io.takari.maven.plugins>
+    <version.io.takari.maven.plugins>1.11.11.jbossorg-1</version.io.takari.maven.plugins>
+    <version.io.takari.maven.plugins.compiler>1.13.5</version.io.takari.maven.plugins.compiler>
```
[the remark this answers is not in the document]
Comment (desmax74) [reply pom.xml]: Ok for the comments, about the PR I'l reopen with tests, the my CLA is already in their db

File: pom.xml [hunk 2]
```diff
@@ -216,7 +216,8 @@
     <gwt.compiler.localWorkers>1</gwt.compiler.localWorkers>
     <!-- property for productisation to know the last released version -->
     <latestReleasedVersionFromThisBranch>notYetReleased</latestReleasedVersionFromThisBranch>
-    <version.io.takari.maven.plugins>1.13.5</version.io.takari.maven.plugins>
+    <version.io.takari.maven.plugins>1.11.11.jbossorg-1</version.io.takari.maven.plugins>
```
Comment (porcelli) [inline pom.xml]: why this `1.11.11.jbossorg-1` version? what if we just don't override it? wasn't supposed to used the already defined one?

Comment (desmax74) [reply pom.xml]: In some jenkins machine (prod) is replaced with the .11.11.jbossorg-1 version, we could leave the 1.13.5 on both and add the comment about why two uses the same version

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by desmax74** [comment]
jenkins test

**Comment by jhrcek** [comment]
@desmax74 please add the comment explaining the difference between those two seemingly redundant properties (one used for testing and override of version in tests + link to the takari PR or something so future people will know what to do).

**Comment by desmax74** [comment]
@jhrcek  comment added

**Pull request merged**